### DISCUSSION TITLE
how do i set

### DISCUSSION THREAD

**Discussion by ayalajw** [arkypita/LaserGRBL#1200] [Q&A]
how do i set for burning different materials?  and is there a chart?

**Comment by belialboy** [comment]
After loading an image and specifying the rendering type, you click on the Next button to create your g-code. In the dialog that then pops up, to the right of the Engraving Speed variable, you'll see a button that looks like an open book. If you click on that you can see the material types and laser types so that you can select the combination that matches your setup. 

If you want to add new materials with your own settings you can use the Grbl->Material DB to create them.
